### DISCUSSION TITLE
我想在電腦上養寵物

### DISCUSSION THREAD

**Discussion by shuxian12** [1101-GS4719-C/Python-Community-GS4719-C#1180] [General]
前幾天在網路上看到一個非常好玩的vscode套件[vscode-pets](https://marketplace.visualstudio.com/items?itemName=tonybaloney.vscode-pets)，下載完之後可以在左選單的檔案總管最下面看到一個小視窗，只要在裡面添加動物，就可以跟他玩，像是可以跟他打招呼，或是丟球給你的寵物玩。

人家的貓貓會後空翻，vscode的貓貓會爬牆給你看 🐱，如果你覺得這些還不夠的話，還有小雞可以養，畢竟現在雞蛋很貴，只是vscode的🐔好像不會下雞蛋，但是他可以咕咕叫。

但我最想介紹的寵物就是那隻暴衝的**霹靂炎上小狗**，他叫做blue，不覺得全身都燃燒的小狗狗很帥嗎。

歡迎在底下告訴我你最喜歡的寵物，如果code常常打不出來，或是bug找不到，又或是覺得打扣太無聊，就趕快去下載吧，沒事玩玩可以降低演算法解不出來的機率！！
![](https://i.imgur.com/Y5wL0fA.png)
![](https://i.imgur.com/e3N2a0i.png)

**Comment by shuxian12** [comment]
我自己寫的，不是chatGPT寫的

**Comment by Lipunpun** [comment]
我之前家裡也有養雞，在電腦裡面養有很多好處，其中雞的排泄物我真的受不了，幾乎每小時都會拉，他又不像是貓狗會定點，所以我認為這個程式對於我這種人十分受用！

**Reply by Brownnnn78** [reply]
我家也有養雞鴨鵝，這裏教各位怎麼區分公雞還是母雞，母雞脖子的毛都會被公雞咬掉，為了生蛋（所以我都不敢吃家裡的蛋，有時候會開到🐤）。

**Reply by shuxian12** [reply]
我外婆家也有養雞，有一次從那邊拿回來的雞蛋每個都是雙蛋黃，感覺超級神奇，而且會發現雙蛋黃的蛋都明顯比較長，但也不會因為這樣所以蛋黃就比較小，但至從那次之後就沒吃過雙蛋黃ㄌＱＱ

**Comment by CCello** [comment]
雖然這個套件已經出了一段時間，但是對我來說是很新鮮的，在處理很複雜的程式碼心情漸漸煩躁時，說不定能起到意想不到的效果!

**Comment by zilinng** [comment]
看到文章之後立刻去下載這個套件，對我這個因為家人不喜歡寵物加上過敏體質一直都沒辦法養狗狗的人來說，在電腦上養寵物真的是一個不錯的選擇，寫作業寫到快崩潰的時候看看可愛的小狗還能轉換心情，對我來說很有用～

**Reply by shuxian12** [reply]
我覺得優點就是每個鐘類的**動物**都有很多不同的顏色，可以滿足大家的喜好，尤其是**狐狸和狗狗**，這些顏色比較多，而且也很可愛！！！

**Comment by curtis1017** [comment]
這讓我想到當我們在使用Colab的時候。我們有時候會因為撰寫程式時寫到有點興致缺缺，所以我發現了一個功能，那就是Colab上面可以跑出柴犬及貓的動態圖像，會在上面跑來跑去。開啟的方法是：開啟Colab，然後點選設定，再來點左方的其他，最後把柯基犬模式和貓咪模式打開後，最後就大功完成囉！

**Comment by DuxtinTai** [comment]
看到這個套件就讓我想到，有很多記帳軟體也會採用類似的方式，透過附加的功能，也許是飼養寵物或是建造城市等，增加使用的樂趣與成就感，藉以協助使用者培養記帳習慣。這類的複合式功能，雖然並不吸引我，卻有不少人買單，確實是一個軟體開發的商機。

**Comment by Lipunpun** [comment]
我在對發票的時候，也有用一個軟體，他也是養寵物讓他跟你有互動，讓我大大的增加對發票的慾望，之前也有玩過摩爾莊園，對於任務中養寵物、種東西的都特別積極，的確是一個很大的商機。

**Comment by Jack-1D** [comment]
看起來好酷喔。只是對我來說好像很容易分心。如果思緒卡住了，看到有隻狗在那邊跑來跑去會感覺很躁。但整體來說，我覺得是個很療育的設計，如果可以設定個簡便的按鈕讓他暫時躲起來可能會更好。

**Comment by shuxian12** [comment]
如果各位是mac的用戶的話，推薦各位可以去載**Desktop Pets**這個套件，他會直接出現在桌面上，也有許多不同的動物或是外星人，你也可以隨便移動他到視窗上面，還附有許多小動畫，或是特別機率出現的動畫，像是突然會出現外星人之類的。
